From 7e2c29b5f96c0301bada42592da5e61509746610 Mon Sep 17 00:00:00 2001
From: Peter Felecan <pfelecan@opencsw.org>
Date: Thu, 26 Sep 2013 09:28:37 +0200
Subject: [PATCH] set manual pages destination

taken from the set of patches made by William Bonnet
---
 xfconf-perl/Makefile.PL.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/xfconf-perl/Makefile.PL.in b/xfconf-perl/Makefile.PL.in
index c16a453..374fb11 100644
--- a/xfconf-perl/Makefile.PL.in
+++ b/xfconf-perl/Makefile.PL.in
@@ -95,6 +95,9 @@ WriteMakefile(
     ABSTRACT_FROM  => 'Xfconf.pm',
     PREREQ_PM      => \%pre_reqs,
     XSPROTOARG     => '-noprototypes',
+    INSTALLSITEMAN1DIR => '/opt/csw/share/man1',
+    INSTALLSITEMAN2DIR => '/opt/csw/share/man2',
+    INSTALLSITEMAN3DIR => '/opt/csw/share/man3',
     $xfconf->get_makefile_vars,
 );
 
-- 
1.8.3.4

